src/components/App.js

```diff
@@ -36,15 +36,18 @@ class Categories extends Component {
   renderCategories() {
     const { navigate } = this.props.navigation;
 
-    const nodes = this.props.categoryData.map(category => (
-      <BoxItem
-        key={category.id}
-        onPress={() => navigate('Detail', { title: category.title, items: category.items })}
-        {...category}
-      />
-    ));
+    let nodes = null;
+    if (this.props.categoryData.length !== 0) {
```

Comment:
is this check necessary?

Reply:
Yeah, that isn't necessary, I'll delete it in next commit 😄!